index.js

```diff
@@ -419,6 +419,55 @@
     return f(y, x);
   });
 
+  //# lift :: Apply f => (a -> b) -> f a -> f b
+  //.
+  //. Promotes a unary function to a function which operates on a Functor.
```

Comment:
Let's use `[Functor][]` and include `[Functor]: https://...` at the bottom of the file.